spirv_cross_parsed_ir.cpp

```diff
@@ -342,6 +342,8 @@ void ParsedIR::set_name(ID id, const string &name)
 {
 	auto &m = meta[id];
 	m.decoration.alias = name;
+	if (ids[id].get_type() == TypeVariable && get<SPIRVariable>(id).remapped_variable)
```

Comment:
This is a little brittle since it relies on calling order of set_remapped_variable_state and set_name. It would be better to defer this check until we iterate over meta_needing_name_fixup.